src/python/pants/task/task.py

```diff
@@ -144,9 +139,8 @@ def alternate_target_roots(cls, options, address_mapper, build_graph):
     """
 
   @classmethod
-  def _prepare(cls, options, round_manager):
+  def invoke_prepare(cls, options, round_manager):
     # Subclasses should not generally need to override this method.
```

Comment:
Given that these methods aren't marked `API: public`, we could probably make this verbiage stronger... or remove it entirely.